submissions/chabuChabu/phpDatabase/viewOrder.php

```diff
@@ -0,0 +1,70 @@
+<?php
+// 1.Connect to database
+require_once("connection.php");
+
+$customer_id  = $_GET['id'];
+
+$savedCookie = 'customerOrder';
+// 2. Check if cookie is empty
+if (empty($_COOKIE[$savedCookie])) {
+
+    include("query.php");
+// 3. Check if cookie has a record
+}else if(!empty($_COOKIE[$savedCookie])) {
+
+    $result = unserialize($_COOKIE[$savedCookie]);
+    foreach ($result as $lkey => $lval){
```

Comment:
From line 16 - 33, I don't understand what's going on

Reply:
@adaugochi Thanks for getting, what action point are giving me here exactly? Do you want me to explain the implementation or is this a bad implementation and i need a to do a new one

Reply:
@adaugochi i understand what you mean now,i will redo the task.thank you